### PR TITLE
Add brace expansion to computed properties

### PR DESCRIPTION
**Pull request by codeNgJon**
Fixes https://github.com/emberjs/guides/issues/1340
Takes [description of property brace expansion](http://emberjs.com/blog/2014/02/12/ember-1-4-0-and-ember-1-5-0-beta-released.html#toc_property-brace-expansion) and adds to the guides.

screen shot 2016-03-28 at 12 29 47 pm

### CODE REVIEW

File: source/localizable/object-model/computed-properties.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ This declares the function to be a computed property, and the arguments tell Emb
...
 
+When you want to depend on a property which belongs to an object, you can setup multiple dependent keys by using brace expansion:
+
+```javascript
+var obj = Ember.Object.extend({
+  baz: {foo: 'BLAMMO', bar: 'BLAZORZ'},
+
+  something: function(){
+    return this.get('baz.foo') + ' ' + this.get('baz.bar');
+  }.property('baz.{foo,bar}')
```
Comment (acorncom) [inline source/localizable/object-model/computed-properties.md]: @codeNgJon Looks great!  One thing we've been shifting toward is an updated syntax that would look more like this:

```
something: Ember.computed('baz.{foo,bar}') {
    return this.get('baz.foo') + ' ' + this.get('baz.bar');
}
```

Would you be able to test that that works (possibly on Ember Twiddle) and update this PR?

**Comment by acorncom** [comment]
Also, we tend to squash commits, so once you've got things worked out nicely, would you do a force push so we only have one commit here?  Thanks!

File: source/localizable/object-model/computed-properties.md (hunk 1 above)
Comment (codeNgJon) [inline source/localizable/object-model/computed-properties.md]: Sounds good. I'll give that a try. Thankss

**Comment by codeNgJon** [comment]
Thanks @acorncom. Added the fiddle link to the description up top.

**Comment by locks** [comment]
@homu r+

**Pull request merged**